docs/guide/known-limitations.md

```diff
@@ -16,7 +16,7 @@ returns a cycle error regardless of whether TRUE or FALSE causes
 a circular reference.
 * There is no data validation against named ranges. For example,
 you can't compare the arguments in a formula like this:
-=IF(firstRange>secondRange; TRUE; FALSE).
+=IF(firstRange>, TRUE, FALSE).
```

Comment:
Is `firstRange>secondRange` changed to `firstRange>` on purpose?

Reply:
It was a mistake. Thanks for spotting it. It's fixed.